x/ibc/23-commitment/merkle/utils.go

```diff
@@ -0,0 +1,34 @@
+package merkle
+
+import (
+	"errors"
+
+	abci "github.com/tendermint/tendermint/abci/types"
+
+	"github.com/cosmos/cosmos-sdk/store/types"
+	ics23 "github.com/cosmos/cosmos-sdk/x/ibc/23-commitment"
+)
+
+func QueryMultiStore(cms types.CommitMultiStore, storeName string, prefix []byte, key []byte) ([]byte, Proof, error) {
```

Comment:
Isn't directly querying the multistore a no-no? Why isn't there a keeper for this? Am I asking the wrong question here?

Reply:
Also can't this be done with the `QueryABCI` function above?